eclair-core/src/main/scala/fr/acinq/eclair/db/sqlite/SqliteChannelsDb.scala

```diff
@@ -31,12 +31,17 @@ class SqliteChannelsDb(sqlite: Connection) extends ChannelsDb {
   import SqliteUtils._
 
   val DB_NAME = "channels"
-  val CURRENT_VERSION = 1
+  val CURRENT_VERSION = 2
 
   using(sqlite.createStatement()) { statement =>
-    require(getVersion(statement, DB_NAME, CURRENT_VERSION) == CURRENT_VERSION) // there is only one version currently deployed
+    getVersion(statement, DB_NAME, CURRENT_VERSION) match {
+      case 1 =>
+        statement.executeUpdate("ALTER TABLE local_channels ADD COLUMN is_closed BOOLEAN NOT NULL DEFAULT 0")
```

Comment:
This is a bit messy, maybe look at https://github.com/ACINQ/eclair/blob/audit-errors/eclair-core/src/main/scala/fr/acinq/eclair/db/sqlite/SqliteAuditDb.scala#L52-L80 for inspiration?